main.go

```diff
@@ -53,8 +65,35 @@ func (p Ploop) GetVolumeName(options map[string]string) flexvolume.Response {
 
 	return flexvolume.Response{
 		Status:     flexvolume.StatusSuccess,
-		VolumeName: options["volumePath"] + "/" + options["volumeId"],
+		VolumeName: p.id(options),
+	}
+}
+
+func prepareVstorage(options map[string]string) (string, error) {
+	v := vstorage.Vstorage{options["clusterName"]}
+	mount, err := v.Mountpoint()
+	if err != nil {
+		return "", err
+	}
+	if mount == "" {
+		// not mounted yet, let's mount
+		if options["clusterPassword"] == "" {
+			return "", errors.New("Failed to find a vstorage cluster mountpoint")
```

Comment:
It is exactly what I'd like to see - we failed to find a vstorage cluster mounted already and user have not specified clusterPassword in order to give us ability to mount it.